src/inspector/processing/stats-processing.service.ts

```diff
@@ -102,34 +109,125 @@ export class StatsProcessingService {
+    const missAttestationsLastEpoch = await this.storage.getValidatorCountByConditionAttestationsLastNEpoch(slot, 1, 'attested = 0');
+    const highIncDelayAttestationsLastEpoch = await this.storage.getValidatorCountByConditionAttestationsLastNEpoch(
+      slot,
+      1,
+      'inclusion_delay > 1',
+    );
+    const invalidHeadAttestationsLastEpoch = await this.storage.getValidatorCountByConditionAttestationsLastNEpoch(
+      slot,
+      1,
+      'valid_head = 0',
+    );
+    const invalidTargetAttestationsLastEpoch = await this.storage.getValidatorCountByConditionAttestationsLastNEpoch(
+      slot,
+      1,
+      'valid_target = 0',
+    );
+    const invalidSourceAttestationsLastEpoch = await this.storage.getValidatorCountByConditionAttestationsLastNEpoch(
+      slot,
+      1,
+      'valid_source = 0',
+    );
+    const missAttestationsLastNEpoch = await this.storage.getValidatorCountByConditionAttestationsLastNEpoch(
+      slot,
+      this.config.get('BAD_ATTESTATION_EPOCHS'),
+      'attested = 0',
+    );
+    const highIncDelayAttestationsLastNEpoch = await this.storage.getValidatorCountByConditionAttestationsLastNEpoch(
+      slot,
+      this.config.get('BAD_ATTESTATION_EPOCHS'),
+      'inclusion_delay > 1',
+    );
+    const invalidHeadAttestationsLastNEpoch = await this.storage.getValidatorCountByConditionAttestationsLastNEpoch(
+      slot,
+      this.config.get('BAD_ATTESTATION_EPOCHS'),
+      'valid_head = 0',
+    );
+    const invalidTargetAttestationsLastNEpoch = await this.storage.getValidatorCountByConditionAttestationsLastNEpoch(
+      slot,
+      this.config.get('BAD_ATTESTATION_EPOCHS'),
+      'valid_target = 0',
+    );
+    const invalidSourceAttestationsLastNEpoch = await this.storage.getValidatorCountByConditionAttestationsLastNEpoch(
+      slot,
+      this.config.get('BAD_ATTESTATION_EPOCHS'),
+      'valid_source = 0',
+    );
+    const highAvgIncDelayAttestationsOfNEpoch = await this.storage.getValidatorCountHighAvgIncDelayAttestationOfNEpochQuery(
       slot,
       this.config.get('BAD_ATTESTATION_EPOCHS'),
     );
     const highRewardMissAttestationsLastNEpoch =
       possibleHighRewardValidators.length > 0
-        ? await this.storage.getValidatorCountWithMissedAttestationsLastNEpoch(
+        ? await this.storage.getValidatorCountByConditionAttestationsLastNEpoch(
             slot,
             this.config.get('BAD_ATTESTATION_EPOCHS'),
+            'attested = 0',
             possibleHighRewardValidators,
           )
         : [];
     operators.forEach((operator) => {
       const missAttestationLast = missAttestationsLastEpoch.find((a) => a.nos_name == operator.name);
       this.prometheus.validatorsCountMissAttestation.set(
         { nos_name: operator.name },
-        missAttestationLast ? missAttestationLast.miss_attestation_count : 0,
+        missAttestationLast ? missAttestationLast.suitable : 0,
```

Comment:
At this point I don't understand what `suitable` means.

Reply:
Just `total` maybe?

Reply:
renamed to `amount`